rego-templates/vuls-slack.rego

```diff
@@ -185,15 +185,17 @@ result = res {
     text:=sprintf("%s%s/%s", [aqua_server, input.registry, input.image])
     urlText :=sprintf("See more: \u003c%s|%s\u003e", [href, text])
 
-    footers := [
-        {
+    footers := by_flag(
```

Comment:
There is a case for slack when "message is too long for display" is shown in thread along with link to aqua server.
in case if no url is configured display "Please configure Aqua server url to see entire message". Add link to Postee documentation